Dockerfile

```
@@ -13,15 +13,48 @@ ENV DEBIAN_FRONTEND noninteractive
 RUN <<EOF
 	set -eux
 	apt-get update
-	apt-get install --no-install-recommends -y curl gnupg python3 python3-pip python3-dev python3-setuptools python3-wheel build-essential git
+	# We need:
+	# * git: for fetching moira and capturing git rev in Meteor artifact
+	# * curl: for the Meteor installer and fetching new apt keys
+	# * gnupg: for installing new apt keys
+	# * python3 et al: for building mediasoup
+	# * comerr-dev et al: for building moira
+	apt-get install --no-install-recommends -y \
+		build-essential \
+		git \
+		curl \
+		gnupg \
+		python3 python3-pip python3-dev python3-setuptools python3-wheel \
+		comerr-dev libkrb5-dev libreadline-dev libhesiod-dev libncurses5-dev autotools-dev
 
 	# Install chromium-browser's dependencies, which should match puppeteer's
 	# dependencies. Note: this will need to be updated when we upgrade to 20.04,
 	# as chromium-browser on 20.04 is a wrapper around a snap package (although
-	# `apt-get satisfy` will make it easier)
-	apt-get install --no-install-recommends -y $(apt-cache depends chromium-browser | sed -ne 's/^ *Depends://p')
+	# apt-get satisfy will make it easier)
+	apt-get install --no-install-recommends -y \$(apt-cache depends chromium-browser | sed -ne 's/^ *Depends://p')
```

At a glance, I'm guessing the build failure about an unexpected `(` is from this `$` -> `\$` conversion here.

```suggestion
	apt-get install --no-install-recommends -y $(apt-cache depends chromium-browser | sed -ne 's/^ *Depends://p')
```